Real pebble tests failing intermittently

[Example failed run here.](https://github.com/canonical/operator/actions/runs/12680641179/job/35342833594)

The failure is:

```
  File "/home/runner/work/operator/operator/test/test_real_pebble.py", line 132, in test_checks_and_health
    assert bad_check.failures == 2
AssertionError: assert 3 == 2
 +  where 3 = CheckInfo(name='bad', level=CheckLevel.READY, status=CheckStatus.DOWN, failures=3, threshold=2, change_id=ChangeID('12')).failures
=========================== short test summary info ============================
FAILED test/test_real_pebble.py::TestRealPebble::test_checks_and_health - AssertionError: assert 3 == 2
```

I assume it's because `test_checks_and_health` has some code in it that's too timing-sensitive. Not sure why it just started happening recently. It's happened a couple of times today, but I don't remember seeing it before that.

--- a/test/test_real_pebble.py
+++ b/test/test_real_pebble.py
@@ -129,7 +129,7 @@ class TestRealPebble:
             time.sleep(0.06)
         else:
             assert False, 'timed out waiting for "bad" check to go down'
-        assert bad_check.failures == 2
+        assert bad_check.failures >= 2
         assert bad_check.threshold == 2
         good_check = next(c for c in checks if c.name == 'good')
         assert good_check.status == pebble.CheckStatus.UP

